src/net_processing.h

```diff
@@ -23,6 +23,7 @@ static const unsigned int DEFAULT_MAX_ORPHAN_TRANSACTIONS = 100;
 static const unsigned int DEFAULT_BLOCK_RECONSTRUCTION_EXTRA_TXN = 100;
 static const bool DEFAULT_PEERBLOOMFILTERS = false;
 static const bool DEFAULT_PEERBLOCKFILTERS = false;
+static const int BANSCORE_THRESHOLD = 100;
```

Comment:
can be moved to the cpp file and made constexpr?

Reply:
> can be moved to the cpp file and made constexpr?

good idea

Reply:
Done, then moved it back to the header file to be able to use it in the unit tests. This allowed for some nice improvements there.